src/templates/finops-hub/modules/keyVault.bicep

```diff
@@ -83,8 +86,8 @@ resource keyVault 'Microsoft.KeyVault/vaults@2023-02-01' = {
     }
     networkAcls: {
       bypass: 'AzureServices'
-      defaultAction: 'Deny'
-  }
+      defaultAction: enablePublicAccess ? 'Allow' : 'Deny'
```

Comment:
nit: Can we keep this set to Allow even if it's not needed?

Reply:
Yes, we need it for for remote hub support when not using PE's.